### PR TITLE
Rename fixes

### PR DESCRIPTION
**Pull request by Jonathan1600**
**What work was done?**
- Describe in detail what's been modified/created

**Why was this work done?**
- We renamed most of the variables for the pages, forms, tables and drop-down menus. 

### Type of change
Please delete options that are not relevant.
- Super necesary . 

Change Status
- Completed, ready to review and merge

### Checklist
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- My code has been reviewed by at least one peer
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- There are no merge conflicts

### CODE REVIEW

**Review by robertjallen — APPROVED** [review]

File: src/components/common/MetricsFilterBar/MetricsFilterBar.js [hunk 1]
```diff
@@ -69,8 +69,11 @@ const MetricsFilterBar = ({
       >
         {programs.map(individualProgram => {
           return (
-            <Option value={individualProgram.id} key={individualProgram.id}>
```
Comment (Monicascz) [inline src/components/common/MetricsFilterBar/MetricsFilterBar.js:72]: Thanks for continuing to tackle the renaming fields list!

File: src/components/forms/AddRecipientForm.js [hunk 2]
```diff
@@ -52,7 +55,7 @@ function AddRecipientForm({ visible, onCreate, onCancel, households }) {
           </Form.Item>
           <Form.Item
             label="Last Name"
-            name="last_name"
+            name="program_last_name"
```
Comment (Monicascz) [inline src/components/forms/AddRecipientForm.js:58]: Should this be recipient_last_name?

**Review by Monicascz — COMMENTED** [review]
Double check that the component src/components/forms/AddRecipientForm has recipient_last_name on line 58 instead of program_last_name as currently written.

**Review by Monicascz — APPROVED** [review]
Thanks for making that change! Nice work on all of the renaming on the frontend! Well done!

**Pull request merged**